### PR TITLE
New package: JWLimited.Crypter version 0.20.1.0

### PR DESCRIPTION
**Pull request by JWLMT88**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/131157)

### CODE REVIEW

**Comment by JWLMT88** [comment]
@microsoft-github-policy-service agree [company="{JW Limited}"]

**Comment by JWLMT88** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

File: manifests/j/JWLimited/Crypter/0.20.1.0/JWLimited.Crypter.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: JWLimited.Crypter
+PackageVersion: 0.20.1.0
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: net8.0-windows10.0.22000.0\crypterv2.exe
+  PortableCommandAlias: n
```
Comment (stephengillie) [inline manifests/j/JWLimited/Crypter/0.20.1.0/JWLimited.Crypter.installer.yaml:10]:
```suggestion
  PortableCommandAlias: n
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.8
```

File: manifests/j/JWLimited/Crypter/0.20.1.0/JWLimited.Crypter.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
```
Comment (stephengillie) [inline manifests/j/JWLimited/Crypter/0.20.1.0/JWLimited.Crypter.installer.yaml:10]:
```suggestion
  PortableCommandAlias: crypter
```
This becomes the symlink name, so please consider something closer to the application name.

**Comment by stephengillie** [comment]
Footnote: The testing VM was consuming 18 GB of RAM.

**Pull request closed without merging**